doc/pset.mediawiki

```diff
@@ -0,0 +1,429 @@
+| The Peg-in Transaction serialized without witnesses.
+|
+| 0
+| 2
+|-
+| Peg-in Transaction Output Proof
+| <tt>PSBT_ELEMENTS_IN_PEG_IN_TXOUT_PROOF = 0x05</tt>
+| None
+| No key data
+| <tt><txoutproof></tt>
+| The transaction output proof for the Peg-in Transaction.
+|
+| 0
+| 2
+|-
+| Peg-in Genesis Hash
+| <tt>PSBT_ELEMENTS_IN_PEG_IN_GENESIS = 0x06</tt>
+| None
+| No key data
+| <tt><hash></tt>
+| The 32 byte genesis hash for the Peg-in Transaction.
+|
+| 0
+| 2
+|-
+| Peg-in Claim Script
+| <tt>PSBT_ELEMENTS_IN_PEG_IN_CLAIM_SCRIPT = 0x07</tt>
+| None
+| No key data
+| <tt><script></tt>
+| The claim script for the Peg-in Transaction.
+|
+| 0
+| 2
+|-
+| Peg-in Value
+| <tt>PSBT_ELEMENTS_IN_PEG_IN_VALUE = 0x08</tt>
+| None
+| No key data
+| <tt><64-bit int></tt>
+| The little endian 64-bit value of the peg-in for the Peg-in Transaction.
+|
+| 0
+| 2
+|-
+| Peg-in Witness
+| <tt>PSBT_ELEMENTS_IN_PEG_IN_WITNESS = 0x09</tt>
+| None
+| No key data
+| <tt><script></tt>
+| The Peg-in witness for the Peg-in Transaction.
+|
+| 0
+| 2
+|-
+| Issuance Inflation Keys Amount
+| <tt>PSBT_ELEMENTS_IN_ISSUANCE_INFLATION_KEYS = 0x0a</tt>
+| None
+| No key data
+| <tt><64-bit int></tt>
+| The value for the inflation keys output to set in this issuance. This is mutually exclusive with <tt>PSBT_ELEMENTS_IN_ISSUANCE_INFLATION_KEYS</tt>.
+|
+| 0
+| 2
+|-
+| Issuance Inflation Keys Amount Commitment
+| <tt>PSBT_ELEMENTS_IN_ISSUANCE_INFLATION_KEYS_COMMITMENT = 0x0b</tt>
+| None
+| No key data
+| <tt><33 byte commitment></tt>
+| The 33 byte commitment to the inflation keys output value in this issuance. This is mutually exclusive with <tt>PSBT_ELEMENTS_IN_ISSUANCE_INFLATION_KEYS</tt>
+|
+| 0
+| 2
+|-
+| Issuance Blinding Nonce
+| <tt>PSBT_ELEMENTS_IN_ISSUANCE_BLINDING_NONCE = 0x0c</tt>
+| None
+| No key data
+| <tt><32 byte blinding nonce</tt>
+| The 32 byte asset blinding nonce. For new assets, must be 0. For reissuances, this is a revelation of the blinding factor for the input.
+|
+| 0
+| 2
+|-
+| Issuance Asset Entropy
+| <tt>PSBT_ELEMENTS_IN_ISSUANCE_ASSET_ENTROPY = 0x0d</tt>
+| None
+| No key data
+| <tt><32 byte entrpy></tt>
+| The 32 byte asset entropy. For new issuances, an arbitrary and optional 32 bytes of no consensus meaning combined used as additional entropy in the asset tag calculation. For reissuances, the original, final entropy used for the asset tag calculation.
+|
+| 0
+| 2
+|}
+
+The currently defined elements per-output proprietary types are as follows:
+
+{|
+! Name
+! <tt><keytype></tt>
+! <tt><keydata></tt>
+! <tt><keydata></tt> Description
+! <tt><valuedata></tt>
+! <tt><valuedata></tt> Description
+! Versions Requiring Inclusion
+! Versions Requiring Exclusion
+! Versions Allowing Inclusion
+|-
+| Value Commitment
+| <tt>PSBT_ELEMENTS_OUT_VALUE_COMMITMENT = 0x01</tt>
+| None
+| No key data
+| <tt><33 byte commitment></tt>
+| The 33 byte Value Commitment for this output. This is mutually exclusive with <tt>PSBT_OUT_VALUE</tt>.
```

Comment:
We should also add a note mentioning that `PSBT_OUT_VALUE` is not mandatory in pset whereas it is in psbt2.

Reply:
Done